.github/workflows/ci.yml

```diff
@@ -282,8 +282,8 @@ jobs:
       env:
         DOCKER_CLI_EXPERIMENTAL: enabled
       run: |
-        wget https://raw.githubusercontent.com/mattermost/mattermost-server/master/server/build/Dockerfile
-        wget https://raw.githubusercontent.com/mattermost/mattermost-server/master/server/build/entrypoint.sh
+        wget https://raw.githubusercontent.com/mattermost/mattermost/master/server/build/Dockerfile
```

Comment:
I did notice that the old paths here still work fine, but I'm good to change them to be consistent